AUTHORS

```diff
@@ -491,18 +531,21 @@ NikolaMandic <mn080202@gmail.com>
 noducks <onemannoducks@gmail.com>
 Nolan Darilek <nolan@thewordnerd.info>
 nzwsch <hi@nzwsch.com>
+O.S. Tezer <ostezer@gmail.com>
 OddBloke <daniel@daniel-watkins.co.uk>
 odk- <github@odkurzacz.org>
 Oguz Bilgic <fisyonet@gmail.com>
 Oh Jinkyun <tintypemolly@gmail.com>
 Ole Reifschneider <mail@ole-reifschneider.de>
 Olivier Gambier <dmp42@users.noreply.github.com>
-O.S. Tezer <ostezer@gmail.com>
 pandrew <letters@paulnotcom.se>
+panticz <mail@konczalski.de>
```

Comment:
these commits were signed off by another person `Sven Dowideit` so not sure what to do here
https://github.com/docker/docker/commits?author=panticz